app/controllers/return-logs-setup.controller.js

```diff
@@ -20,7 +22,14 @@ async function setup(request, h) {
   const { returnLogId } = request.query
   const session = await InitiateSessionService.go(returnLogId)
 
-  return h.redirect(`/system/return-logs/setup/${session.id}/how-to-edit`)
+  return h.redirect(`/system/return-logs/setup/${session.id}/start`)
+}
+
+async function start(request, h) {
+  const { sessionId } = request.params
+  const pageData = await StartService.go(sessionId)
+
+  return h.view('return-logs/setup/start.njk', { activeNavBar: 'search', ...pageData })
```

Comment:
Not a massive issue but to keep it consistent inside the controller maybe the activeNavBar property can go inside the StartService, like how the Received function looks

```suggestion
  return h.view('return-logs/setup/start.njk', { ...pageData })
```